### PR TITLE
[3] fix(twap): don't request singleOrders data for just mined orders

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

For some reason, `composableCowContract.singleOrders` result from multicall might return falsy value.
It happens when a TWAP order transits from Signing to Open state.
To fix this problem, we don't check `singleOrders` for orders that were mined recently `(<30s)`

  # To Test

1. Create a TWAP order
- [ ] It should NOT become `Cancelled` after tx mining

### CODE REVIEW

**Review by anxolin — COMMENTED** [review]
Are you sure you are not covering sth up?

Could it be that you get `boolean`, `syncing`, or `error` for some time? (if that's the case, you might be able to handle it

```ts
export interface CallState {
    readonly valid: boolean;
    readonly result: CallStateResult | undefined;
    readonly loading: boolean;
    readonly syncing: boolean;
    readonly error: boolean;
}
```

**Comment by shoom3301** [comment]
@anxolin your question made me double-check and I've found a problem in my logic.
I should have checked `info.safeData.safeTxParams.isExecuted` before checking `existingOrder.status` - it's a root of the problem. Thank you!

The fix: https://github.com/cowprotocol/cowswap/pull/2761/commits/fe21509f3e458c6d4040a978616fe7db83e98069

File: src/modules/twap/updaters/TwapOrdersUpdater.tsx [hunk 1]
```diff
@@ -75,3 +48,39 @@ export function TwapOrdersUpdater(props: {
...
+        return {
+          id: getConditionalOrderId(data.conditionalOrderParams),
+          orderStruct: parseTwapOrderStruct(data.conditionalOrderParams.staticInput),
+          safeData: data,
+        }
+      } catch (e) {
+        return null
+      }
+    })
+    .filter(isTruthy)
```
Comment (alfetopito) [inline src/modules/twap/updaters/TwapOrdersUpdater.tsx:65]: Nitpick: reduce?

**Review by alfetopito — APPROVED** [review]

File: src/modules/twap/updaters/TwapOrdersUpdater.tsx (hunk 1 above)
Comment (shoom3301) [reply src/modules/twap/updaters/TwapOrdersUpdater.tsx:65]: Makes sense, thanks!

**Review by elena-zh — COMMENTED** [review]
Hey @shoom3301 , the order does not go to 'Cancelling' state. 
However, as a nitpick, on the Swap and Limit orders we show 'Order presigned' pop-up when an order goes to the Open state. 
I have not noticed this pop-up here. 
Let me know please if I need to create an issue for this. Thanks!

**Comment by shoom3301** [comment]
@elena-zh I'm not aware of this popup. Could you should a screenshot of it please?

**Comment by elena-zh** [comment]
> @elena-zh I'm not aware of this popup. Could you should a screenshot of it please?

Here it is 
image

**Review by elena-zh — APPROVED** [review]
I have created #2787 task for the presigned pop-up

**Comment by anxolin** [comment]
> @anxolin your question made me double-check and I've found a problem in my logic.
I should have checked info.safeData.safeTxParams.isExecuted before checking existingOrder.status - it's a root of the problem. Thank you!

Thanks for checking!

**Pull request merged**